### PR TITLE
Replace Travis CI with GitHub Actions

### LINKED ISSUE #1686: [FEATURE]: Move to GitHub Actions
GitHub Actions is faster, easier and better to use, IMHO.
I think it would be a good idea to replace Travis CI with GitHub Actions.

What do you think about this? 🙂

### PR DESCRIPTION
**Pull request by Panquesito7**
Closes #1686.
Things added/changed:

- Switch from Travis CI to GitHub Actions.

### CODE REVIEW

**Comment by cubbimew** [comment]
hm, this gets "/bin/sh: 1: hunspell: not found"

**Comment by Panquesito7** [comment]
Ping?

**Comment by cubbimew** [comment]
> Ping?

it failed to run hunspell even though you mention it the env section, any ideas why?
I see some other people use "SPELLCHECKER_PREFER_HUNSPELL: true" there

**Comment by Panquesito7** [comment]
> any ideas why?

No idea, it should work properly.

> I see some other people use "SPELLCHECKER_PREFER_HUNSPELL: true" there

That's for [AppVeyor](https://github.com/marketplace/appveyor) I think? I've tried that, though: doesn't work yet.

**Comment by Panquesito7** [comment]
[Hunspell](https://github.com/hunspell/hunspell) installation and use should be working properly now. 🙂

**Comment by cubbimew** [comment]
awesome, thanks

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    steps:
+      - uses: actions/checkout@master
+     
+      - name: Install Hunspell
+        run: |
+           sudo apt-get update
+           sudo apt-get install hunspell
+           
+      - name: Run cd scripts; make -k
+        run: cd scripts; make -k
```
Comment (jwakely) [inline .github/workflows/build.yml:17]: Why not just use `make -C scripts -k` and not need to `cd` and then the next step isn't needed.

Comment (Panquesito7) [reply .github/workflows/build.yml:17]: I prefer to keep it the way it is. It's the same algorithm as in `.travis.yml`. 🙂

**Pull request merged**